.travis.yml

```diff
@@ -20,4 +20,4 @@ matrix:
     - rvm: jruby-head
   fast_finish: true
 bundler_args: "--jobs 4"
-before_install: ruby -e "Dir['{,spec/**/}Gemfile.lock'].each { |f| File.write(f, File.read(f).split('BUNDLED WITH').first)}"
+before_install: ruby -e "Dir['{,spec/**/}Gemfile.lock'].each { |f| version = File.read(f).split(/BUNDLED WITH\n   /).last; system('gem', 'install', 'bundler', '-v', version)}"
```

Comment:
don't think this is necessary, travis should have a modern bundler by default

Reply:
... and if not maybe best to do a `gem install bundler` here so it runs with latest

Reply:
Sure! I think we can do `gem install bundler` :+1:.

Reply:
I ended up removing this change, because you were right that `travis` has a modern bundler by default.